bd_spot_wrapper/spot_wrapper/spot.py

```diff
@@ -306,7 +314,9 @@ def block_until_arm_arrives(self, cmd_id, timeout_sec=5):
             self.command_client, cmd_id, timeout_sec=timeout_sec
         )
 
-    def get_image_responses(self, sources, quality=None, pixel_format=None):
+    def get_image_responses(
+        self, sources, quality=None, pixel_format=None
+    ):  # returns 'google.protobuf.pyext._message.RepeatedCompositeContainer'
```

Comment:
This message of "Returns ..." would be better placed in the docstring of the function

Reply:
addressed.